apps/meteor/ee/app/livechat-enterprise/server/lib/requestPdfTranscript.ts

```diff
@@ -0,0 +1,44 @@
+import { OmnichannelTranscript, QueueWorker } from '@rocket.chat/core-services';
+import type { AtLeast, IOmnichannelRoom } from '@rocket.chat/core-typings';
+import { LivechatRooms } from '@rocket.chat/models';
+
+import { logger } from './logger';
+
+const serviceName = 'omnichannel-transcript' as const;
+export const requestPdfTranscript = async (
+	room: AtLeast<IOmnichannelRoom, '_id' | 'open' | 'v' | 'pdfTranscriptRequested'> | null,
+	requestedBy: string,
+): Promise<void> => {
+	if (!room) {
```

Comment:
Should we really check this? 🤔

Reply:
It's to remove the `null` type from the room object. I copied it over from the service without much looking, maybe we don't need it anymore